freud/box.pyx

```diff
@@ -300,13 +304,18 @@ cdef class Box:
         vecs = np.asarray(vecs)
         flatten = vecs.ndim == 1
         vecs = np.atleast_2d(vecs)
-        vecs = freud.util._convert_array(vecs, shape=(None, 3)).copy()
+        vecs = freud.util._convert_array(vecs, shape=(None, 3))
+        out = freud.util._convert_array(out, shape=vecs.shape)
 
         cdef const float[:, ::1] l_points = vecs
         cdef unsigned int Np = l_points.shape[0]
-        self.thisptr.wrap(<vec3[float]*> &l_points[0, 0], Np)
+        cdef const float[:, ::1] l_out = out
 
-        return np.squeeze(vecs) if flatten else vecs
+        self.thisptr.wrap(<vec3[float]*> &l_points[0, 0],
+                          Np, <vec3[float]*> &l_out[0, 0])
+        if flatten:
+            out = np.squeeze(out)
+        return out
```

Comment:
Write this as one line so that this method matches the style of similar methods.
```suggestion
        return np.squeeze(out) if flatten else out
```